content/en/security/application_security/security_signals/users_explorer.md

```diff
@@ -0,0 +1,174 @@
+
+1. Click **Block** in the user's row, and choose a blocking duration.
+2. In **Select Security Responses**, select **Block with Datadog's Library**.
+   
+   Permanently or temporarily blocked authenticated users are added to the [Denylist][6]. Manage the list on the Datadog [Denylist][7] page.
+
+
+## Compare and block multiple users
+
+When you select two or more users, you can use the **Compare and Block** button to compare datapoints across compromised users.
+
+When you click **Compare and Block**, several metrics are displayed in **Block selected users**. These metrics help you detect whether you are dealing with **a single attacker**, **a larger coordinated campaign**, or **widespread credential exposure**.
+
+Here's a breakdown of the benefits of comparing each datapoint across two (or more) compromised users.
+
+### Users per ASN
+
+**Benefit:** Identifies if compromised accounts are accessed from the same internet service provider or network operator.
+
+If multiple users are compromised from the same ASN (Autonomous System Number), it could suggest:
+  * A botnet operating from that network.
+  * A VPN or proxy exit node.
+  * A targeted attack from a specific ISP region (for example, a university or corporate network).
+
+### Users per User Agent
+
+**Benefit:** Reveals if the attacker is using automated tools or a specific browser configuration.
+
+Matching user agents could mean:
+  * Same malware toolkit (for example, info-stealer like RedLine or Raccoon).
+  * Use of headless browsers or scripts.
+  * Credential stuffing using a shared toolset.
+
+### Users per Location (Geo-IP)
+
+**Benefit:** Exposes if access attempts are coming from the same country or region.
+
+If multiple users are being accessed from the same location (especially foreign or unexpected ones), it:
+  * Points to targeted regional attacks.
+  * Helps detect suspicious login anomalies (for example, impossible travel).
+  * May indicate TOR exit nodes or VPN clusters.
+
+### Users per Domain
+
+**Benefit:** Groups affected users by email domain, which typically represents their organization.
+
+Benefits include:
+  * Identifying organization-wide breaches.
+  * Highlighting phishing campaigns targeting a specific company or domain.
+  * Supporting incident response and alert prioritization.
+
+### Threat Intel Category
+
+**Benefit:** Shows the associated [Threat Intelligence Category][5].
+
+Comparing helps you:
+  * Understand the vector of compromise.
+  * Prioritize responses (for example, `hosting_proxy` might require different mitigation than `malware`).
+
+### Threat Intel Intention
+
+**Benefit:** Clarifies **why the attacker stole the credentials** (fraud, espionage, resale, lateral movement, etc.).
+
+Matching intentions reveal:
+  * Common attacker **goals**.
+  * Whether your users were part of a **targeted vs opportunistic** campaign.
+  * Whether you should expect **further action**, like BEC (business email compromise) or internal movement.
```

Comment:
All the threat intel information provide insights about the attacker IPs reputation.

Reply:
Aligned, flagged IPs aren't necessarily _bad_, they're just known to sometimes be misused (often without the user knowing in case of `residential_proxy`) and worth paying closer attention to